news/2024/2024-01-16-mappers-guild-more-priority-quests.md

```diff
@@ -0,0 +1,104 @@
+
+Mappers can [earn rewards](/wiki/Community/Mappers_Guild#rewards) for completing mapping-related tasks, like ranking maps, participating in contests, and collaborating on [quests](/wiki/Community/Mappers_Guild#quests) (which is what this article is about!).
+
+The Mappers' Guild is open to mappers of any skill level, including those without ranked maps! See [the system's website](https://mappersguild.com/) or the [osu! Discord server](https://discord.com/invite/ppy)'s `#mappers-guild` channel for details!
+
+## What are priority quests?
+
+[Just a few months ago](https://osu.ppy.sh/home/news/2023-08-29-changes-to-the-mappers-guild), the Mappers' Guild introduced a new way to get involved: [**priority quests**](https://mappersguild.com/missions). These are an alternative to [ordinary Mappers' Guild quests](https://mappersguild.com/quests) with a few key differences:
+
+- **Anyone can participate in priority quests.** Unlike normal quests, you don't need to claim priority quests in advance.
+- **Quests are tiered by difficulty:**
+  - Tier 1 ![bronze icon](/wiki/shared/news/2023-08-29-changes-to-the-mappers-guild/bronze-small.png) (easiest, most common, and lowest reward)
+  - Tier 2 ![silver icon](/wiki/shared/news/2023-08-29-changes-to-the-mappers-guild/silver-small.png)
+  - Tier 3 ![gold icon](/wiki/shared/news/2023-08-29-changes-to-the-mappers-guild/gold-small.png)
+  - Tier 4 ![platinum icon](/wiki/shared/news/2023-08-29-changes-to-the-mappers-guild/platinum-small.png) (hardest, least common, and highest reward)
+- Depending on the quest's completion conditions, **multiple users can earn prizes for the same quest**.
+- **Priority quests don't last forever.** Maps can be added to priority quests until a deadline is met or a beatmap completes the quest.
+
+By completing priority quests, you can earn some special rewards too:
+
+- All the usual [Mappers' Guild rewards](/wiki/Community/Mappers_Guild#rewards) for ranking beatmaps
+- A [**customised keychain**](/wiki/shared/news/2023-08-29-changes-to-the-mappers-guild/keychain.jpg) based on a pattern in your map
+- A profile badge upon completing 3 priority quests
+
+![Priority quest badge](/wiki/shared/news/2023-08-29-changes-to-the-mappers-guild/questbadge.png)
+
+## New priority quests
+
+**6** priority quests have been completed so far. Another **6** have just been added to the [priority quest listing](https://mappersguild.com/missions)!
+
+Unless otherwise stated, each quest ends on **April 30, 2024**.
+
+Here's a quick overview of each quest:
+
+### [Underappreciated Artists v2](https://mappersguild.com/missions?id=65a3451cf3b11461108e6fa7)
+
+Some of osu!'s Featured Artists haven't seen much love recently. Create and rank a map from any artist who hasn't received a ranked map since their announcement on osu!. (Relevant artists are listed on [the quest page](https://mappersguild.com/missions?id=65a3451cf3b11461108e6fa7)!)
```

Comment:
```suggestion
Some of osu!'s Featured Artists haven't seen much love recently. Create and rank a map from any artist who hasn't received a Ranked map since their announcement on osu!. (Relevant artists are listed on [the quest page](https://mappersguild.com/missions?id=65a3451cf3b11461108e6fa7)!)
```